runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessTile/ProcessTileTagList/ProcessTileTagList.styles.ts

```diff
@@ -45,37 +46,38 @@ export const TagBox = styled.div
     transition: 0.3s;
 `);
 
-export const DividerBox = styled.div
-    <{ $isExpanded: boolean; }>(({ theme, $isExpanded }) => `
+export const DividerBox = styled.div`
     margin-top: 3px;
     display: flex;
     justify-content: center;
     align-items: center;
     width: 100%;
     height: 10px;
-    background-color: ${$isExpanded ? theme.palette.background.paper : 'transparent'};
-`);
+`;
 
-export const DividerAction = styled.div(({ theme }) => `
+export const DividerAction = styled.div
+    <{ $isExpanded: boolean; }>(({ theme, $isExpanded }) => `
     display: flex;
     justify-content: center;
     align-items: center;
     height: 14px;
     width: 70px;
     border-radius: 10px;
     transition: 0.3s;
+    background-color: ${$isExpanded ? theme.palette.grey[200] : 'transparent'};
```

Comment:
you should rather change display: block/none conditionally by isExpanded. If this causes flicking try to figure sth out, maybe combine conditional background-color with conditional z-index